docs/image-transfer/ExportPipelines/azuredeploy.json

```diff
@@ -95,10 +104,29 @@
         "target": {
           "type": "[variables('targetType')]",
           "uri": "[parameters('targetUri')]",
-          "keyVaultUri": "[parameters('keyVaultUri')]"
+          "keyVaultUri": "[concat(reference(resourceId('Microsoft.KeyVault/vaults', parameters('KeyVaultName')), '2019-09-01').vaultUri, 'secrets/', parameters('sasTokenSecretName'))]"
```

Comment:
@bangbingsyb does this work if existing key vaults are creating an in API version other than `2019-09-01`?

Reply:
My understanding is "reference" template function is just a "GET" request to the resource using a given API version. So as long as "2019-09-01" is still a supported key vault API version, this will work. The API version that was used to create the key vault is not relevant here. If the key vault was created via "2016-09-01", we should still be able to "GET" the vault via "2019-09-01", right?

Reply:
I see, verified with a kv in another version and it looks good.